version-data/data.json

```diff
@@ -9,18 +9,18 @@
       },
       {
-        "address": "0x58676340f458b36997608672be6548c92ce50714",
+        "address": "0xcafea63ad9028fa022bf07eed9adb4dccc6c9bbc",
         "code": "CL",
         "smartContractCode": "CL",
         "contractName": "claim1",
```

Comment:
We should see where this property is used and try to update it to the same name that we use in smart-contracts.

Reply:
This isn't used anywhere, but I updated the names everywhere for clarity